config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java

```diff
@@ -74,7 +78,11 @@
      * @return history config info
      */
     @GetMapping
-    public ConfigHistoryInfo getConfigHistoryInfo(@RequestParam("nid") Long nid) {
+    @Secured(action = ActionTypes.READ, parser = ConfigResourceParser.class)
+    public ConfigHistoryInfo getConfigHistoryInfo(@RequestParam("dataId") String dataId, // for @Secured
```

Comment:
Please do not change the paramters

Reply:
persistService find detailConfigHistory add params dataId, group, tenant